packages/bot/src/constants/const.js

```diff
@@ -42,17 +42,24 @@ const config = {
             [translate('Error'), 'error'],
```

Comment:
@aaron-binary please don't name it const. its better if you use something more specefic